### PR TITLE
padding-top when logged out

### PR DESCRIPTION
**Pull request by stonecrusher**
https://i.imgur.com/o0uhyju.png

edit: **add image inline**
padding-top

### CODE REVIEW

**Comment by the-j0k3r** [comment]
Confirmed issue, this is likely upstream issue since it seems like its comming from padding.scss with selector `.py-3`

Unchecking those fixes this issue, see gif.

padding-top

Need to see what else maybe going on.

Edit: this is going on without this style installed or even GHD, so definitely looks like upstream.

capture

**Comment by the-j0k3r** [comment]
Workaround:

```
/* ==UserStyle==
@name           HACK GitHub-FixedHeader#12
@namespace      StylishThemes
@version        1.0.0
@description    A temp fix for https://github.com/StylishThemes/GitHub-FixedHeader/pull/12
@author         Some lil' Elf
==/UserStyle== */

@-moz-document domain("github.com") {
  .position-relative .header-logged-out {
		padding: 5px !important;
  }
}
```
wasnt sure what padding the logged in has and 0 seemed too much due to search bar when focused.

Alternatively, merge this.

File: github-fixed-header.user.css [hunk 1]
```diff
@@ -18,6 +18,9 @@
   .is-stuck {
     padding-top: var(--height) !important;
   }
+  body.logged-out {
+    padding-top: 89px !important;
```
Comment (the-j0k3r) [inline github-fixed-header.user.css:22]: You meant `padding-top: -89px !important;` else it goes like

capture here

**Comment by the-j0k3r** [comment]
Actually my first two comments dont apply to this issue, its something related only. Sorry for noise.

**Comment by stonecrusher** [comment]
Your noise lead me to another small issue with my fix, so it actually helped :-)
Btw there is nothing like "negative padding".
Upstream issue label can be removed.

**Comment by xt0rted** [comment]
The package.json should be updated too.

https://github.com/StylishThemes/GitHub-FixedHeader/blob/6cb8a2fcda5293e75ec5c03d50201f33de93a594/package.json#L4

**Pull request merged**